playbooks/common/openshift-master/config.yml

```diff
@@ -224,6 +224,21 @@
   roles:
   - pods
   - os_env_extras
+  - role: nickhammond.logrotate
+    logrotate_scripts:
+        - name: syslog
+          path: "/var/log/cron
+                 \n/var/log/maillog
+                 \n/var/log/messages
+                 \n/var/log/secure
+                 \n/var/log/spooler \n"
+          options:
+            - daily
+            - rotate 7
+            - compress
+            - sharedscripts
+          scripts:
```

Comment:
We should probably make these settings configurable with these being the defaults...  Also make the logrotate role optional as well, since there will be some users who will wish to manage logrotate on their own.